### ISSUE TITLE
Scrapers down across federal jurisdictions and oral arguments?

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#2003]
I've gotten a few reports that something is messed up with our scrapers. People are complaining about missing oral arguments and opinions.

Gotta dig in. Seems like a major outage. Maybe related to micro-service transition, but I'm not sure how.

**Comment by flooie** [comment]
We’ll shucks

**Comment by mlissner** [comment]
OK, this is fixed, ± some lingering cleanup. For history's sake, the postmortem is that:

1. We switched to microservices, and had a call that was failing.
2. Unfortunately, that was inside a try/except block that caught all exceptions and converted them to simple warnings.
    - The reason for this is because scrapers fail a lot, so we long ago decided to mostly ignore their noise except when working on them.

To fix this, we:

1. Fixed the microservice call.
2. Are working to add tests and probably assertions to prevent bad code in the future.
3. Removed the broad try/except block. Now errors will go to Sentry and we can triage from there.

The scrapers are back up and we'll be cleaning things up, but that should do it.
